src/js/util/api/api.js

```diff
@@ -0,0 +1,64 @@
+const SERVER_URL = "http://localhost:3001";
+
+const api = {
+  get: async (url) => {
+    try {
+      const res = await fetch(`${SERVER_URL}/api/category${url ? url : ""}`);
+
+      if (!res) {
+        throw new Error("서버 에러");
+      }
+
+      return await res.json();
+    } catch (e) {
+      throw new Error(e);
+    }
+  },
+  post: async (url, data) => {
+    try {
+      const res = await fetch(`${SERVER_URL}/api/category${url ? url : ""}`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json;charset=utf-8",
+        },
+        body: JSON.stringify(data),
```

Comment:
data가 없을 경우를 대비해 예외처리 하기. 삼항연산자를 사용하지 않고 default parameter를 사용해도 괜찮을듯
```suggestion
        body: data ? JSON.stringify(data) : null,
```